api/models/series.go

```diff
@@ -77,7 +79,7 @@ func (smp SeriesMetaProperties) Export() SeriesMetaPropertiesExport {
 	}
 }
 
-// Merge merges SeriesMeta b into a
+// Merge merges SeriesMeta b into a and returns the modified a
```

Comment:
```suggestion
// Merge merges SeriesMeta 'b' into 'a' and returns the modified 'a'
```

Reply:
Let's follow the conventions here. See e.g. https://godoc.org/bytes#Equal

Reply:
Sounds good.